cli/consumer_command.go

```diff
@@ -195,6 +195,7 @@ func configureConsumerCommand(app commandHost) {
 	consSub.Arg("consumer", "Consumer name").StringVar(&c.consumer)
 	consSub.Flag("ack", "Acknowledge received message").Default("true").BoolVar(&c.ack)
 	consSub.Flag("raw", "Show only the message").Short('r').UnNegatableBoolVar(&c.raw)
+	consSub.Flag("deliver-group", "Deliver group of the consumer").StringVar(&c.deliveryGroup)
```

Comment:
Isn't the deliver group inferred from the `ConsumerInfo`, which makes this argument unused?

Reply:
Do you mean if we fetch the consumer info we could grab this value?

Reply:
That would be good - but time perhaps better spent on “nats sub” to support binding and behaviours like that via the normal client. 

These consumer sub and pull commands predate us  having clients so I am inclined to deprecate them and keep just nats sub.

Reply:
> Do you mean if we fetch the consumer info we could grab this value?

Correct me if I'm wrong, but isn't the `ConsumerInfo` [fetched from the API](https://github.com/bruth/natscli/blob/c935f5afbe877e918a95c49e53832bd34befbb9f/cli/consumer_command.go#L1441) already?
If I understand the code correctly the attribute this flag sets is only used in the `create` and `edit` sub-commands, and therefore unused in the `sub` in this context.